src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,72 @@
+import "./Sidebar.css";
+
+import { useState } from "react";
+
+import { Icon, IconNames } from "..";
+
+interface Item {
+  icon: IconNames;
+  text: string;
+}
+interface Section {
+  heading: string;
+  items: Item[];
+}
+interface SidebarProps {
+  isVisible: boolean;
+  sections: Section[];
+}
+export const Sidebar = ({ ...props }: SidebarProps) => {
+  const { sections } = props;
+  const [isCollapsed, setCollapsed] = useState(false);
+
+  const toggleVisibility = () => {
+    setCollapsed(!isCollapsed);
+  };
+
+  return (
+    <div className="sidebar">
+      <div className={`sidebar-full ${isCollapsed ? "sidebar-collapsed" : ""}`}>
+        <div className="menu_header">
+          <button
+            type="button"
+            className="usa-button usa-button--outline usa-button--inverse"
+            onClick={toggleVisibility}
+          >
+            {!isCollapsed ? "<<" : ">>"}
```

Comment:
Should these arrows be icons instead of text?

Reply:
The button will have to change once the button components are created.